rclcpp/src/rclcpp/context.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 Open Source Robotics Foundation, Inc.
+// Copyright 2015-2020 Open Source Robotics Foundation, Inc.
```

Comment:
Why? I mean it is allowed, but it seems done so inconsistently through our packages, even within this pull request. Is it useful?

Reply:
I agree we don't consistently do this, but I remember having read that we should.
If you think it's a problem, I can revert the change.

Reply:
I don't think it is necessary, but I also don't care if you leave it. It's just you don't add it to other files you touched in this pull request, so I don't know when and when not to add it.